kodilla-testing/src/main/java/com/kodilla/testing/forum/shape/Shape.java

```diff
@@ -0,0 +1,8 @@
+package com.kodilla.testing.forum.shape;
+
+interface Shape {
+
+    String getShapeName();
+
+    double getField();
+}
```

Comment:
poniewaz w kolekcji trzymasz obiekty Shape, to kazda klasa implementujaca ten interfejsc powinna moec zaimplementowana wlasna metode equals i hasCode